CONTRIBUTING.md

```diff
@@ -3,4 +3,4 @@
    this file would be a symlink.
 -->
 
-[Contribution guidelines can be found in `doc/`](https://semgrep.dev/docs/contributing/how-to-contribute/).
+[Contribution guidelines can be found in here](https://semgrep.dev/docs/contributing/contributing/).
```

Comment:
```suggestion
[Semgrep's contribution guidelines](https://semgrep.dev/docs/contributing/contributing/)
are hosted on our documentation website.
```

I try to avoid using the word "here" in links and instead write out the name of what the link points to, as it leads to better accessibility (e.g. with screen readers), SEO, and machine-parsing.

Reply:
Good, words such as "here" or "this" should not be used in links. @underyx